network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreController.java

```diff
@@ -225,6 +225,21 @@ public ResponseEntity<Void> deleteSubstation(@Parameter(description = "Network I
         return ResponseEntity.ok().build();
     }
```

Comment:
I think we should not keep the previous endpoints?
For create/update it's always a list. We should do the same for consistency.

Reply:
There anyway not efficient and are sometimes not adapted. For example, we often don't want to delete one switch but plenty instead.